src/main.c

```diff
@@ -368,8 +368,6 @@ static void library_main_helper(struct libargs_s *args) {
         case GET_PRINTABLE_AMOUNT:
             // ensure result is zero if an exception is thrown (compatibility breaking, disabled
             // until LL is ready)
-            // args->get_printable_amount->result = 0;
```

Comment:
this is actually a single block of comments

Reply:
OK. Fixed in 56e4a3cf9abef7d5c84a3d5d13ccd600f318ec09.